scripts/helm.sh

```diff
@@ -34,3 +34,12 @@ clean_cluster() {
     return 0
   fi
 }
+
+# install the kubernetes helm binary.
+install_helm() {
```

Comment:
we might consider baking helm into the Dockerfile like [helmc is](https://github.com/deis/e2e-runner/blob/master/Dockerfile#L24-L25) since we're going to be moving entirely over to official helm in the near future...